.ko.yaml

```diff
@@ -0,0 +1,4 @@
+# Use :nonroot base image for all containers
```

Comment:
This repo only defines one `cmd/` package, right? Does `defaultBaseImage` accomplish anything, since it's just overridden below?

Reply:
Correct. 

The test image should be using this base image(`test/upload-test-images.sh`), but we dont have any particular use case in test images to use this base image. Also just checked their sizes, the default one is 3 times larger than alpine image. @itsmurugappan I think we can simply default to alpine base image here.

Reply:
The test images(present and future ones) will refer this base image. A non root image will give the flexibility to run in contributors clusters in which non root images are not allowed. Maybe I can change it to 'gcr.io/distroless/static:nonroot' which is used in all the knative repos.